Description: remove gcc ftemplate-depth-300 flag
 it can use a lot of memory, and default value works on debian
Forwarded: not-needed, https://github.com/mapnik/mapnik/commit/b01b21f24
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2013-10-12
--- a/SConstruct
+++ b/SConstruct
@@ -1664,7 +1664,7 @@
         if not env['SUNCC']:
 
             # Common flags for CXX compiler.
-            common_cxx_flags = '-ansi -Wall %s %s -ftemplate-depth-300 ' % (env['WARNING_CXXFLAGS'], pthread)
+            common_cxx_flags = '-ansi -Wall %s %s ' % (env['WARNING_CXXFLAGS'], pthread)
 
             # https://github.com/mapnik/mapnik/issues/1835
             if sys.platform == 'darwin' and env['CXX'] == 'g++':
